charts/oes/templates/oesdb-deployment.yaml

```diff
@@ -1,11 +1,16 @@
-apiVersion: extensions/v1beta1
-kind: Deployment
+apiVersion: apps/v1
+kind: StatefulSet
 metadata:
   labels:
     app: db-opsmx
   name: db-opsmx
 spec:
+  podManagementPolicy: {{ .Values.opsmxdb.podManagementPolicy }}
```

Comment:
Specially for things like this, we should have defaults that we think will work 90% of the cases and the user should provide override if they want something different.

Reply:
Shall we keep default value as "Parallel" instead of "OrderedReady" ?